### ISSUE TITLE
[필독] [공지사항] 안녕하세요. 2023 구강이미지 합성데이터 헬스케어 AI 경진대회 운영팀입니다. 

### ISSUE THREAD

**Issue by bab-korea** [bab-korea/healthcare-ai-contest#112]
안녕하세요. 2023 구강이미지 합성데이터 헬스케어 AI 경진대회 운영팀입니다.

참고하실 수 있는 자주 묻는 질문 리스트 올려드립니다.

1. 최종 결과물(source_file, performance_metric_file)은 배정시간에 내는것일까요?
- 배정시간이 아니더라도 제출은 가능합니다.

2. Source dir에는 (1) 경진대회에 수행한 모든 코드에 대한 zip 파일 (2) 최종 inference 코드 (3) 모든 function(trian, test, dataloader 등)을 포함한 코드 중에 어떤것을 업로드하는것일까요?
- 모든걸 포함한 파일입니다.

3. Weight file은 source dir에 업로드하면 될까요?
- Weight는 업로드하지 않습니다.

4. 보고서 작성할때는 서버에 접속이 불가능한 것일까요? 보고서 쓸때 접속못하은건지 여쭤봅니다.
- 보고서 작성시에는 서버에 접속이 불가합니다

5. F1스코어 소수점 이하 자리수가 참가자별 2개의 형태(16자리, 4자리)로 나타나는 것으로 보이는데 4자리 스코어는 어떻게 판단해야하는 건지 알아봐주세요.
- 4자리, 16자리 구분없이 정렬해서 순위 매기면 될 것 같습니다.

6. 사전학습모델은 사용가능한가요?
- 사전학습모델은 운영지침상 사용이 불가합니다.

7. 산출물 txt에 f1 score만 작성되도 된다고 하셨는데
검증을 위한 (파일명 / Ground Truths / predict)은 필요 없는건가요?
그리고 weight 파일을 통해 성능이 재현되는지에 대한 테스트도 같이 진행하시는지 궁금합니다.
- 검증단이 테스트 검증을 진행하지만, Ground Truth, predict는 제출하지 않아도 됩니다.

감사합니다.

**Comment by wheresmadog** [comment]
weight를 업로드 하지 않는데, reproduce는 어떻게 진행되나요?
해당 weight를 훈련 할 당시의 configuration을 따로 남기더라도 seed 관리에 따라서는 정확한 재현이 안 될 수도 있을텐데요.

**Comment by Junhyuk93** [comment]
3. weight 파일을 업로드 하지않았는데 어떻게 재현하여 검증하나요?  재현성을 위한 train 코드나, inference 코드에 대한 배치 파일도 만들어야 하는 것 아닌가요?
 
4. GPU 할당 시간을 고정시키셨는데 보고서 작성시에 서버 접속이 불가하다 하시면 사실상 대회 마무리시간보다 훨씬 이전에 팀별로 종료되는 시간이 달라지는 것 아닌가요?

5. 소수점 자리에 따라서 동률인 점수가 나올 가능성이 높아지는데 상관없다고 말씀하시는게 무슨 말인지 이해가 안됩니다.

6. pretrained weights 사용이 불가한 점을 사전에 미리 고지하셨어야 대회 참가자분들이 오해의 소지없이 모델을 구축하는 것 아닌가요? 공모전 홈페이지에는 활용AI모델(다른 모델 사용 가능) 이라고만 작성 해두셨던데요.
